### PR TITLE
Updated Introduction

### PR DESCRIPTION
**Pull request by hannestschofenig**

### CODE REVIEW

File: draft-ietf-rats-msg-wrap.md [hunk 1]
```diff
@@ -92,21 +92,30 @@ This document also defines a corresponding CBOR tag, as well as JSON Web Tokens
...
 
 It is desirable to reuse any typing information associated with the messages
 across such protocol boundaries in order to minimize the cost associated with
-type registrations and maximize interoperability.
+type registrations and maximize interoperability. With the CMW format described
+in this document protocol designers do not need to update protocol specifications
+to support different conceptual messages. This approach reduces the implementation
+effort for developers to support different attestation technologies. For example,
+an implementer of a Relying Party application does not need to parse
+attestation related conceptual messages, such as different Evidence formats,
```
Comment (thomas-fossati) [inline draft-ietf-rats-msg-wrap.md:116]:
```suggestion
attestation-related conceptual messages, such as different Evidence formats,
```

File: draft-ietf-rats-msg-wrap.md [hunk 2]
```diff
@@ -121,8 +130,8 @@ These encapsulation formats are designed to be:
   registration cost across many different usage scenarios.
 
 A protocol designer could use these formats, for example, to convey
-evidence, endorsements or reference values in certificates and CRLs
-extensions ({{DICE-arch}}), to embed attestation results or evidence as
+Evidence, Endorsements and reference values in certificates and CRLs
+extensions ({{DICE-arch}}), to embed Attestation Results or Evidence as
 first class authentication credentials in TLS handshake messages
```
Comment (thomas-fossati) [inline draft-ietf-rats-msg-wrap.md:135]:
```suggestion
extensions ({{DICE-arch}}), to embed Attestation Results or Evidence as
first-class authentication credentials in TLS handshake messages
```

File: draft-ietf-rats-msg-wrap.md [hunk 3]
```diff
@@ -92,21 +92,30 @@ This document also defines a corresponding CBOR tag, as well as JSON Web Tokens
...
-Attestation Results for Secure Interactions (AR4SI) {{-rats-ar4si}} payloads in
-"passport" mode would go first from Verifier to Attester and then, at a later
-point in time and over a different channel, from Attester to Relying Party.
+
+- Attestation Results for Secure Interactions (AR4SI) {{-rats-ar4si}} payloads in
+"passport" mode would be sent by the Verifier to the Attester and then, at a later
+point in time and over a different channel, from the Attester to the Relying Party.
 
 It is desirable to reuse any typing information associated with the messages
 across such protocol boundaries in order to minimize the cost associated with
```
Comment (thomas-fossati) [inline draft-ietf-rats-msg-wrap.md:110]:
```suggestion
across such protocol boundaries to minimize the cost associated with
```

**Review by thomas-fossati — APPROVED** [review]
good stuff, thanks!

While the iron is hot, I have a suggestion for removing some awkwardness from L123-130:

---

The following encapsulation formats have been specifically designed to possess the following characteristics:
- They are self-describing, which means that they can convey precise typing information without relying on the framing provided by the embedding protocol or the storage system.
- They are based on media types {{-media-types}}, which allows the cost of their registration to be spread across numerous usage scenarios.

---

File: draft-ietf-rats-msg-wrap.md [hunk 4]
```diff
@@ -92,21 +92,30 @@ This document also defines a corresponding CBOR tag, as well as JSON Web Tokens
...
 Each conceptual message can have multiple claims encoding and serialization
-formats ({{Section 9 of -rats-arch}}).
-Throughout their lifetime, RATS conceptual messages are typically transported
-over different protocols.
-For example, EAT {{-rats-eat}} evidence in a "background check" topological
+formats ({{Section 9 of -rats-arch}}). Throughout their lifetime, RATS
+conceptual messages are typically transported over different protocols.
+For example, 
+
+- EAT {{-rats-eat}} evidence in a "background check" topological
```
Comment (hannestschofenig) [inline draft-ietf-rats-msg-wrap.md:101]:
```suggestion
- EAT {{-rats-eat}} Evidence in a "background check" topological
```

File: draft-ietf-rats-msg-wrap.md [hunk 5]
```diff
@@ -92,40 +92,46 @@ This document also defines a corresponding CBOR tag, as well as JSON Web Tokens
...
 
-* Based on media types {{-media-types}} - which allows amortising their
-  registration cost across many different usage scenarios.
+* They are based on media types {{-media-types}}, which allows the cost of their registration to be spread across numerous usage scenarios.
 
 A protocol designer could use these formats, for example, to convey
-evidence, endorsements or reference values in certificates and CRLs
-extensions ({{DICE-arch}}), to embed attestation results or evidence as
-first class authentication credentials in TLS handshake messages
+Evidence, Endorsements and reference values in certificates and CRLs
```
Comment (nedmsmith) [inline draft-ietf-rats-msg-wrap.md:130]: > Evidence, Endorsements and reference values in certificates and CRLs

Evidence, Endorsements and Reference Values in certificates and CRLs

**Review by nedmsmith — COMMENTED** [review]
Reference Values should be capitalized to follow the convention consistently.

**Pull request merged**